scripts/dailyevents.gd

```diff
@@ -20,16 +20,18 @@ func play(stage = 0):
 		person.lust = -rand_range(5,10)
 		person.lewdness += rand_range(3,5)
 		person.lastsexday = globals.resources.day
-		globals.resources.mana += rand_range(2,3)
-		###---Added by Expansion---### Races Expanded
-		if person.race.find("Dark Elf") >= 0:
-		###---Expansion End---###
-			globals.resources.mana += 1
+		if !succubuscheck(person,1,1): #ralphC - also indented below
```

Comment:
`succubuscheck` is called 12 times in this file but never declared.

Reply:
must have missed a copy/paste - will add func

Reply:
fixed for next commit